### DISCUSSION TITLE
Welcome to ai-privacy-toolkit Discussions!

### DISCUSSION THREAD

**Discussion by abigailgold** [IBM/ai-privacy-toolkit#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
 
  To get started, add a comment below (please introduce yourself).

**Comment by Ataullah128** [comment]
---> 19 from apt.utils.dataset_utils import get_iris_dataset_np, get_adult_dataset_pd, get_german_credit_dataset_pd
using windows 10 
and almost all libraries given in requirements .txt
still this issue is not solved

**Reply by abigailgold** [reply]
@Ataullah128 Can you please post the full stack trace and let us know which version of apt you're using? Also, an issue would be a better place for addressing specific issues with the code. Thanks!
